src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```

Reply:
"If I remove "static", I will get an error saying "non-static method getRandomBall() cannot be referenced from a static context"."

Reply:
Because `Random` and `ColorSupplier` should also be non-static